Tests/SwerverTests/TestData.swift

```diff
@@ -8,4 +9,29 @@ class TestData {
     static func validUrlTwo() -> URL {
         return URL(string: "/foo")!
     }
-}
\ No newline at end of file
+    
+    static func simpleGetUrl() -> URL {
+        return URL(string: Resource.test_get.rawValue)!
+    }
+    
+    static func getWithBodyUrl() -> URL {
+        return URL(string: Resource.test_head.rawValue)!
+    }
+    
+    static func methodOptionsUrl() -> URL {
+        return URL(string: Resource.test_option.rawValue)!
+    }
+
+    static func notFoundUrl() -> URL {
+        return URL(string: "/not_here")!
+    }
+    
+    static func buildResponse() -> HttpResponse {
```

Comment:
I'm not sure I like this abstraction. For most of the tests I saw using it, it hid details that the test cared about.

Reply:
will fix that